Scripts/ProtoWrappers.py

```diff
@@ -0,0 +1,902 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
+import sys
```

Comment:
Protobuf schemas have a pretty simple syntax, and we only use a narrow subset of it.

In the interest of pragmatism, this script only handles the schema syntax that we use and expects us to format the schema using our conventions.